automation/check-patch.sh

```diff
@@ -245,11 +245,10 @@ run() {
 }
 
 check_generated_code() {
-    make generate
-    if [[ -n "$(git status --porcelain)" ]] ;
+    if [[ -n "$(gofmt -l ./tests)" ]] ;
```

Comment:
Don't assume that `gofmt` is installed, rather do the check in the build container.